.travis.yml

```diff
@@ -2,7 +2,7 @@ language: php
 php:
   - 7.2
   - 7.3
-  - 7.4snapshot
+  - 7.4
   - nightly
 matrix:
   include:
```

Comment:
In order to build as in other PHP versions, I think php 7.4 with lowest and highest dependencies should be added

Reply:
Could you provide your the suggested code snippets to resolve this issue you mention?

Reply:
Append to build `matrix`
```
    - php: 7.4
      env: dependencies=lowest
    - php: 7.4
      env: dependencies=highest
```